docs/_static/css/custom.css

```diff
@@ -1,21 +1,39 @@
 /* ReadTheDocs theme colors */
 
-
 .wy-nav-top { background-color: #f07e44 }
+.wy-nav-content {
+    max-width: 1200px;
+}
 .wy-side-nav-search { background-color: transparent }
 .wy-side-nav-search input[type="text"] { border-width: 0 }
 
 
 /* Custom classes */
 
-
 .small { font-size:40% }
 .smaller, .pr, .noteversion { font-size:70% }
 .noteversion::after { content: "/" }
 
 
 /* Code: literals and links */
```

Comment:
Now this comment doesn’t apply to what’s below it. Please move it to its correct position and add a comment for the “rubric” CSS rules (what do they do?)

Reply:
Will fix it... Right...

Reply:
Fixed it: https://github.com/theislab/scanpy/pull/960/commits/3e33d8d2119c2167faccabece4a58d0f556c6ef5